wemake_python_styleguide/logic/system.py

```diff
@@ -3,11 +3,11 @@
 from wemake_python_styleguide.constants import WINDOWS_OS
 
 
-def is_executable_file(filename: str) -> bool:
+def is_executable_file(filename: str) -> bool:  # pragma: not-win
     """Checks if a file is executable."""
     return os.access(filename, os.X_OK)
 
 
-def is_windows() -> bool:
+def is_windows() -> bool:  # pragma: win
```

Comment:
This function executes on both linux and win code, so no reason to ignore it.

Reply:
Done